prover/protocol/wizard/verifier.go

```diff
@@ -73,15 +73,21 @@ type VerifierRuntime struct {
 	// round. The first entry is the initial state, the final entry is the final
 	// state.
 	FiatShamirHistory [][2][]field.Element
+	// the parent run time; used in the distributed setting,
```

Comment:
As a note. 

I am ok with that for now. In my current work, I have thought of a more generic change to the verifier and prover runtime allowing you to store arbitrary data